agent/tools/write_enriched_sprint_example.md

```
@@ -0,0 +1,1060 @@
+
+## Test contract
+
+All test functions are `async def`. None have decorators. All take `client` from conftest as a parameter; routes that require auth additionally take `auth_headers`. Multi-volunteer scenarios construct the second volunteer via `db_session` (the `test_volunteer` fixture provides only one).
+
+### `tests/test_locations.py` (7 tests)
+
+| Test | Action | Asserts |
+|---|---|---|
+| `test_create_location_returns_201(client)` | POST `/locations` `{"name": "Main", "address": "1 Test St"}` | 201, body has `id`, `name == "Main"` |
+| `test_list_locations_empty(client)` | GET `/locations` | 200, `body == []` |
+| `test_list_locations_nonempty(client, test_location)` | GET `/locations` | 200, `len(body) == 1`, body[0]["id"] == str(test_location.id) |
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: ""id""

(MD052, reference-links-images)